news/870.plugin

```diff
@@ -0,0 +1 @@
+Switch Ax parser to use new Hydra command line syntax
```

Comment:
Is this diff doing both? why?
if not, what is this news fragment doing here?

Reply:
Hey :) This is a WIP PR, to check all tests pass on the CI. Once #912 lands, I will rebase and request review.